AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -533,9 +452,13 @@ TH2D RayTraceCorrelator::GetInterferometricMap(
             double dt = it_delays[iterBin];
 
             p0 = int((dt - xVals[0]) / dx);
-            if (p0 < 0) p0 = 0;
-            if (p0 >= numPoints) p0 = numPoints - 2;
-            corrVal = (yVals[p0 + 1] - yVals[p0]) * ((dt - xVals[p0]) / (xVals[p0 + 1]-xVals[p0])) + yVals[p0];
+            if(p0<0 || p0>=numPoints){
```

Comment:
We can make this a single comparison if it seems useful:
`if(abs(p0-(numPoints-1.)/2.) > (numPoints-1.)/2.)`

Reply:
I feel like the double if seems more transparent. Tho yours is probably faster (tho idk by how much). But yours requires some math operations. I can try.

Reply:
Definitely less transparent. My feeling is the logic is the most expensive part but that could be wrong. But if this is benchmarked fast enough then we can skip it.

Reply:
Okay, I checked it. The mathy way is slower. The double comparison is 0.0254s/event, while the math one is 0.0268s/event. So doing the math must be slower than the double comparison.